suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -451,14 +452,16 @@ export const countUniqueCoins = (accounts: Account[]) => {
 export const enhanceTokens = (tokens: Account['tokens']) => {
     if (!tokens) return [];
 
-    return tokens
-        .filter(t => t.symbol && t.balance)
-        .map(t => ({
+    return tokens.map(t => {
+        const symbol = formatTokenSymbol(t.symbol || t.contract);
+
+        return {
             ...t,
-            name: t.name || t.symbol,
-            symbol: t.symbol!.toLowerCase(),
-            balance: formatAmount(t.balance!, t.decimals),
-        }));
+            name: t.name || symbol,
+            symbol: symbol.toLowerCase(),
```

Comment:
nitpick:

You have this in the description:
```
ignore nonsense toLowerCase, there is another issue for that. Symbols can be mixed case
```

So why do you use `toLowerCase` here or am I missing something?

Reply:
I expressed myself poorly. 

The lowercase does not make sense but I am keeping it there just to be sure that it does not break anything, but it will be removed soon